inst/bench/bench.R

```diff
@@ -0,0 +1,120 @@
+library(data.table)
```

Comment:
I meant in the folder `./bench` and not in `./inst/bench` as everything in `./inst` is included in the installed R package